### DISCUSSION TITLE
join function doesn't work with an array of strings

### DISCUSSION THREAD

**Discussion by michaelfecher** [community/community#38604] [Actions]
Hi,

I wanted to use the join function in a composite action with an array of two strings and a dot (.) separator like shown in the following image:
![image](https://user-images.githubusercontent.com/13333848/200683721-5ce9627a-3381-4bde-adb3-8030ccd41bd3.png)

However, it fails with this error:
```
Unexpected symbol: '['. Located at position 6 within expression: join([steps.transformStageName.outputs.lowercase, inputs.apexDomain], '.')
```
Thanks for helping me out.

**Comment by david-engelmann** [comment]
Did you mean to use `github.event.inputs`?

**Reply by michaelfecher** [reply]
No, I wanted to use the join function within a composite action.
The `inputs` section of the composite action has the `apexDomain` parameter defined.

**Comment by david-engelmann** [comment]
I would try to `join(fromJson([ ...`
